src/container/Movies.js

```diff
@@ -1,14 +1,53 @@
 import React,{Component} from "react"
 import axios from 'axios'
+import Moviescss from "./Moviescss.css"
 import Movie_list from "../components/Movie_list"
+import Spinner from '../common/Spinner'
+import MovieFilters from '../components/MovieFilters'
 
 class Movies extends Component{
 	constructor(props){
 		super(props)
 	this.state={
-		movies_list:[]
+		movies_list:[],
+		search_value:'',
+		message: '',
+		isloading: false,
+		filter:{
+			is4: false,
+			is3: false,
```

Comment:
The variables needs to be changed. I am not getting what is the purpose of this variable here.